lib/fetch-server.js

```diff
@@ -154,21 +141,21 @@ export class FetchServer {
 
   debugLog({url, params, response}) {
     if (this.debug) {
-      const responseObjectPromise = response.then(r => r.clone());
-      const responseBodyPromise = responseObjectPromise.then(clonedResponse => clonedResponse.json());
+      const responseObjectPromise = response.then((r) => r.clone());
+      const responseBodyPromise = responseObjectPromise.then((clonedResponse) => clonedResponse.json());
       Promise.all([responseObjectPromise, responseBodyPromise]).then(([response, responseBody]) => {
         console.groupCollapsed(`[fetch-server] ${url}`);
-          console.groupCollapsed('Params');
```

Comment:
Ooh, where did these linter comments come from? anyhow, since this is explicitly a debug-logging method, let's use `eslint-disable` around this block so it doesn't complain about the console statements

Reply:
Those are github action smart enough to stream linting warning/error to comment~